docs/src/docs/asciidoc/routing.adoc

```diff
@@ -0,0 +1,83 @@
+[[routing]]
```

Comment:
Note that this folder has completely outdated content and is not linked or built when building the addons artifact. See #82. That work alone could be a reason to "revive" the asciidoc guide though...